Doc/c-api/unicode.rst

```diff
@@ -85,6 +85,16 @@ access to internal read-only data of Unicode objects:
    subtype.  This function always succeeds.
 
 
+.. c:function:: int PyUnicode_READY(PyObject *o)
+
+   Returns ``0``. This API is kept only for backward compatibility.
+
+   .. versionadded:: 3.3
+
+   .. deprecated:: 3.10
+      This API do nothing since Python 3.12. Please remove code using this function.
```

Comment:
Huge number of code using this API.
https://github.com/hpyproject/top4000-pypi-packages/search?q=PyUnicode_READY